js/app/articleHTMLRenderer.js

```diff
@@ -179,21 +68,15 @@ var ArticleHTMLRenderer = new Knowledge.Class({
         // "Legacy" content, including most wiki sources and also 2.6 Prensa Libre content,
         // is templated and styled on the client.
 
-        if (model.is_server_templated) {
```

Comment:
I think it might make sense to move this logic around a little bit. Since you still have `_get_html()` in this code anyway, you could remove the `is_server_template` argument from `eknr_renderer_render_content()`, and rename that method to `eknr_renderer_render_content_legacy()`. Then put the test for `is_server_templated` directly in `render()`.

Reply:
Yeah, I see your point here. Will do.